tests/dir.props

```diff
@@ -114,6 +114,11 @@
     <PackageVersionArg>/p:CoreClrPackageVersion=$(VersionToRestore)</PackageVersionArg>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' != ''">$(__RuntimeId)</RuntimeIdentifiersOverwrite>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' == ''">win-$(__BuildArch)</RuntimeIdentifiersOverwrite>
```

Comment:
@weshaggard just updated this, the property is now in tests/src/dir.props & uses TestNugetRuntimeID, like TargetRID does.